### ISSUE TITLE
Translation file for Dutch texts

### ISSUE THREAD

**Issue by CasparTheBridge** [jonasbkarlsson/ev_smart_charging#296]
**Is your feature request related to a problem? Please describe.**
Looks like I can't do a pull request. I've created a Dutch translation for the translation files.
 
**Describe the solution you'd like**
Would be nice if I can also translate the texts of the charging entity etc.

**Describe alternatives you've considered**
Upload it myself as a pull request.

**Additional context**
None

[nl.json](https://github.com/user-attachments/files/16346614/nl.json)

**Comment by jonasbkarlsson** [comment]
Thanks for the translation. You should be able to do a pull request. Others have been able to to that. Do you want to try again?

Also, what did you mean by "translate the texts of the charging entity"? Do mean the name of the entities, such as `sensor.ev_smart_charging_charging`?

**Comment by jonasbkarlsson** [comment]
Do you want to provide translation for the following?

```
ENTITY_NAME_CHARGING_SENSOR = "Charging"
ENTITY_NAME_STATUS_SENSOR = "Status"
ENTITY_NAME_ACTIVE_SWITCH = "Smart charging activated"
ENTITY_NAME_APPLY_LIMIT_SWITCH = "Apply price limit"
ENTITY_NAME_CONTINUOUS_SWITCH = "Continuous charging preferred"
ENTITY_NAME_EV_CONNECTED_SWITCH = "EV connected"
ENTITY_NAME_KEEP_ON_SWITCH = "Keep charger on"
ENTITY_NAME_OPPORTUNISTIC_SWITCH = "Opportunistic charging"
ENTITY_NAME_LOW_PRICE_CHARGING_SWITCH = "Low price charging"
ENTITY_NAME_LOW_SOC_CHARGING_SWITCH = "Low SOC charging"
ENTITY_NAME_START_BUTTON = "Manually start charging"
ENTITY_NAME_STOP_BUTTON = "Manually stop charging"
ENTITY_NAME_CONF_PCT_PER_HOUR_NUMBER = "Charging speed"
ENTITY_NAME_CONF_MAX_PRICE_NUMBER = "Electricity price limit"
ENTITY_NAME_CONF_OPPORTUNISTIC_LEVEL_NUMBER = "Opportunistic level"
ENTITY_NAME_CONF_LOW_PRICE_CHARGING_NUMBER = "Low price charging level"
ENTITY_NAME_CONF_LOW_SOC_CHARGING_NUMBER = "Low SOC charging level"
ENTITY_NAME_CONF_MIN_SOC_NUMBER = "Minimum EV SOC"
ENTITY_NAME_CONF_START_HOUR = "Charge start time"
ENTITY_NAME_CONF_READY_HOUR = "Charge completion time"
```

**Comment by jonasbkarlsson** [comment]
@CasparTheBridge, I have added support for translation in the branch [feature.translation](https://github.com/jonasbkarlsson/ev_smart_charging/tree/feature.translation). Also adding translation for entity information (names, states, attributes).

Can you provide the Dutch translations for the entity information, see [this file](https://github.com/jonasbkarlsson/ev_smart_charging/blob/feature.translation/custom_components/ev_smart_charging/translations/nl.json)?

**Comment by CasparTheBridge** [comment]
Thank you very much!

I see I tried to do the initial pull request in the wrong way (and I see I now did the pull request in the wrong way not linked it to the feature.translatation. Should I correct this or can you merge these two?

**Comment by jonasbkarlsson** [comment]
Thank you! I copied the first manually, and the second I have now merged into feature.translation.

**Comment by hmmbob** [comment]
This one can be closed now that #308 has been merged, right?

**Comment by jonasbkarlsson** [comment]
Included in [v2.0.0](https://github.com/jonasbkarlsson/ev_smart_charging/releases/tag/v2.0.0).
